src/table/cells/__tests__/handle-key-press.spec.js

```diff
@@ -96,6 +101,49 @@ describe('handle-key-press', () => {
     });
   });
 
+  describe('moveFocus', () => {
```

Comment:
Now this is redundant. It is both tested for `handleKeyDown` and `headHandleKeyDown` in the same way. You can remove it. Sorry for the back and forth, I thought it was easier to mock `moveFocus`

Reply:
Fixed. Thanks